profile.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quiz App Profile</title>
+    <link rel="stylesheet" href="./styles/index.css" />
+  </head>
+  <body>
+    <h1>Profile</h1>
+    <main>
+      <section>
+        <img src="" alt="" width="200" height="150" />
```

Comment:
for semantic html i think it is better to add a `figure` tag to wrap the `img `
if this shall be indeed a circle maybe it is better to set width and height at the same value

Reply:
oh thank you for the advise, you are totally right! 😊